cpp/arcticdb/version/version_store_api.hpp

```diff
@@ -92,7 +93,8 @@ class PythonVersionStore : public LocalVersionedEngine {
         const py::object &norm,
         const py::object & user_meta,
         bool upsert,
-        bool prune_previous_versions);
+        bool prune_previous_versions,
+        bool ensure_sorted);
 
     VersionedItem update(
```

Comment:
Am I right in assuming there's no `ensure_sorted` flag for `update` because it is effectively always true?

Reply:
correct. As the error message says:  When calling update on sorted data, the new data must be sorted. ArcticDB relies on Pandas to detect if data is sorted - you can call DataFrame.index.is_monotonic_increasing on your input DataFrame to see if Pandas believes the data to be sorted.

There is no point to update otherwise considering our logic